ts/store/reducers/wallet/transactions.ts

```diff
@@ -34,6 +34,17 @@ export const getTransactions = (state: GlobalState) =>
       values(txs).filter(_ => _ !== undefined) as ReadonlyArray<Transaction>
   );
 
+export const getWalletTransactionsCreator = (idWallet: number) => (
+  state: GlobalState
+) =>
+  pot.map(
+    state.wallet.transactions.transactions,
```

Comment:
you may map on the result of `getTransactions` to avoid repeating the call on `values` and the filter on `undefined` values

Reply:
You mean to get getTransactions result and apply and additional .filter with the walletId condition?

Reply:
yes exactly